client/packages/coldchain/src/Equipment/DetailView/Tabs/Summary.tsx

```diff
@@ -281,6 +282,12 @@ export const Summary = ({ draft, onChange, locations }: SummaryProps) => {
               fullWidth
             />
           </Row>
+          <Row label={t('label.needs-replacement')}>
+            <Checkbox
+              checked={draft.needsReplacement as boolean}
```

Comment:
I wondered about this - which casts as bool at the same time
```suggestion
              checked={Boolean(draft.needsReplacement)}
```